scalameta/common/shared/src/main/scala/org/scalameta/internal/MacroHelpers.scala

```diff
@@ -70,6 +70,7 @@ trait MacroHelpers extends DebugFinder
   lazy val IndexOutOfBoundsException = hygienicRef[IndexOutOfBoundsException]
   lazy val IteratorClass = tq"_root_.scala.collection.Iterator"
   lazy val SeqClass = tq"_root_.scala.collection.immutable.Seq"
```

Comment:
I had that PR in mind when I chose List, but if you prefer I can use `Seq` and let you guys handle the refactor.